### PR TITLE
Add Vivid Money (Bank/Fintech)

### PR DESCRIPTION
**Pull request by moriarty-jim**

### CODE REVIEW

File: companies/vivid-money.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "slug": "vivid-money",
+    "relevant-countries": [
+        "de"
```
Comment (baltpeter) [inline companies/vivid-money.json:4]: According to the country selector in the footer, they also target a few other countries. We should add those as well.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

File: companies/vivid-money.json (hunk 1 above)
Comment (moriarty-jim) [reply companies/vivid-money.json]: sure, thanks for your feedback!

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**